mlir/include/mlir-c/Dialect/SparseTensor.h

```diff
@@ -26,20 +26,20 @@ MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(SparseTensor, sparse_tensor);
 /// If updating, keep them in sync and update the static_assert in the impl
 /// file.
 enum MlirSparseTensorLevelType {
```

Comment:
I am not sure what is the default bitwidth for C enum, is it prone to overflow ?

Reply:
There is

enum class XXX : uint64_t

for that purpose, but that may have interesting different requirements here and there

Reply:
Good point! I looked into this and I found that "The choice of type is implementation-defined, but shall be capable of representing the values of all the members of the enumeration." 
[https://stackoverflow.com/questions/17045661/are-there-any-guarantees-on-the-representation-of-large-enum-values](url)

Reply:
did you mean enum XXX : uint64_t (enum class would raise errors)? it would raise this `warning: enumeration types with a fixed underlying type are a Clang extension [-Wfixed-enum-extension] enum MlirSparseTensorLevelType : uint64_t {`. Is that okay?